.github/workflows/ci-locks.yml

```diff
@@ -56,12 +56,19 @@ jobs:
           pixi workspace export conda-explicit-spec --environment ${{ env.NAME }} --frozen --ignore-pypi-errors requirements
           pixi workspace export conda-environment --environment ${{ env.NAME }} requirements/mo-pack.yml
 
+      - name: "generate token"
+        uses: tibdex/github-app-token@3beb63f4bd073e61482598c45c71c1019b59b73a
```

Comment:
This is now provided by an official action from GitHub:

https://github.com/actions/create-github-app-token

Reply:
Also followed-up with https://github.com/SciTools/workflows/pull/125